<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DFRRiskAnalyse</class>
 <widget class="QDialog" name="DFRRiskAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/Top.qrc">
    <normaloff>:/new/wl/Logo_ico.png</normaloff>:/new/wl/Logo_ico.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>8</y>
     <width>731</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>1</y>
      <width>731</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabInputDescribe">
     <attribute name="title">
      <string> 输入选项描叙</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxInput">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPC8">
     <attribute name="title">
      <string>PC8布局图</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="taBOM">
     <attribute name="title">
      <string>BOM表数据</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidgetBOM"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>731</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>360</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>DFR规则检查结果查看</string>
     </property>
     <widget class="QTableWidget" name="tableWidgetResult">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>16</y>
        <width>355</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>367</x>
       <y>5</y>
       <width>360</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>DFR规则检查详细结果查看</string>
     </property>
     <widget class="QTableWidget" name="tableWidgetResultDetail">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>18</y>
        <width>355</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbStartCheck">
     <property name="geometry">
      <rect>
       <x>462</x>
       <y>236</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>启动规则检查</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSubmitResult">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>236</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>提交当前分析结果</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBrdData">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>236</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查看Brd数据</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/Top.qrc"/>
 </resources>
 <connections/>
</ui>
